template-processors/base/bin/wrapper.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# shellcheck disable=SC1090
```

Comment:
please don't disable this globally for whole script, but just for single lines where it happens